packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -339,4 +339,87 @@ describe("calcite-list", () => {
     expect(await listItemOne.getProperty("selected")).toBe(false);
     expect(await list.getProperty("selectedItems")).toHaveLength(0);
   });
+
+  describe.only("keyboard navigation", () => {
+    async function evaluateActiveElement(page: E2EPage, activeElementSelector: string): Promise<void> {
```

Comment:
I think you could use the [`isElementFocused` test util](https://github.com/Esri/calcite-components/blob/91b2a1b92d49cdd573650952ee09971c59bd1649/packages/calcite-components/src/tests/utils.ts#L291-L290) instead.